README.md

```diff
@@ -18,6 +18,30 @@ git clone https://github.com/redhat-et/copilot-ops.git
 # Build the binary
 make build
 ```
+To install the command line tool, complete the following steps:
+
+```console
+# Create bin directory
+mkdir -p ${HOME}/bin
+
+# Save binary to bin directory (run from copilot-ops directory)
+cp copilot-ops ${HOME}/bin
+
+# Make binary executable 
+chmod 755 ${HOME}/bin/copilot-ops
```

Comment:
This part happens by default when running `make build` or downloading the binary, so we can omit it

Reply:
line removed